hw3/01-routing.js

```diff
@@ -41,9 +41,59 @@ const server = http.createServer((req, res) => {
     res.write(`<h1>Exercise 01</h1>`);
     res.write(`<ul> ${routeResults} </ul>`);
     res.end();
+  } 
+  else if (req.url === '/welcome') {
+    let routeResults = getRoutes();
```

Comment:
the result of `getRoutes()` is not required/used in this loop as well as the loops below.